.github/workflows/check.yml

```diff
@@ -68,12 +69,14 @@ jobs:
           cargo +${{ matrix.rust-toolchain }} check $BUILD_TYPE --all-targets --features ci
 
       - name: Run tests and determine coverage
+        env:
+          RUST_LOG: trace
```

Comment:
Do you need this?

Reply:
We do need to set the env variable, so debug-output-producing code is getting coverage-tested. But we can also set that on each run like before, in case that is what you meant?